packages/workspace/src/command-line/format.ts

```diff
@@ -61,43 +53,56 @@ export function format(command: 'check' | 'write', args: yargs.Arguments) {
   }
 }
 
-function getPatterns(args: NxArgs & { libsAndApps: boolean; _: string[] }) {
-  const allFilesPattern = [MATCH_ALL_PATTERN];
+function getPatterns(
+  args: NxArgs & { libsAndApps: boolean; _: string[] },
+  supportedExtensions: string[],
```

Comment:
I think this should be handled within this function.

Reply:
Yeah, totally 👍🏻